docs/AccountBridge.md

```diff
@@ -279,6 +285,28 @@ broadcast({
 
 It returns an **optimistic** `Operation` that this transaction is likely to create in the future.
 
+### `estimateMaxSpendable`
+
+```js
+estimateMaxSpendable({
+  account: AccountLike,
+  parentAccount?: ?Account,
+  transaction?: ?Transaction
+}): Promise<BigNumber>;
+```
+
+is an heuristic that provide the **estimated** max amount that can be set to a send.
+
+This is often the balance minus the fees, but there are exceptions that depends between coins (reserve, burn, frozen part of the balance,...).
+
+By "estimated", it means this is not necessarily correct and there might be a delta that is +- the actual reality (meaning trying to send with this amount can either exceed the spendable or leave some dust). It should not be used to do an actual SEND MAX (there is `useAllAmount` for this) but it can be used for an informative UI or also for "dry run" approaches (try a transaction with that, and refine).
+
+The strategy of implementations should be to prefer pessimistic estimation rather than potentially impossible transaction, in other words implementation should consider the worse case scenario (like sending all UTXOs to a legacy cost more fees to it have less spendable)
```

Comment:
```suggestion
The strategy of implementations should be to prefer pessimistic estimation rather than a potentially impossible transaction, in other words, an implementation should consider the worst-case scenario (like sending all UTXOs to a legacy address has higher fees resulting in a lower max spendable)
```